packages/integrations/tailwind/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@astrojs/tailwind",
+  "description": "Tailwind + Astro Integrations",
+  "version": "0.0.1",
+  "type": "module",
+  "types": "./dist/index.d.ts",
+  "author": "withastro",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/withastro/astro.git",
+    "directory": "packages/integrations/tailwind"
+  },
+  "bugs": "https://github.com/withastro/astro/issues",
+  "homepage": "https://astro.build",
+  "exports": {
+    ".": "./dist/index.js",
+    "./base.css": "./base.css",
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "astro-scripts build \"src/**/*.ts\" && tsc",
+    "dev": "astro-scripts dev \"src/**/*.ts\""
+  },
+  "dependencies": {
+    "tailwindcss": "^3.0.23",
+    "autoprefixer": "^10.4.2",
```

Comment:
This is needed to stop pnpm and others from complaining. Otherwise, you'd need to add them as deps of your own project, but right now this integration is all about not forcing you to add your own postcss.config.js, your own tailwind.config.js, etc.

There's probably some work to do to keep improving these integrations post-merge. Like using your own config file if it does exists, or merging the two, etc.